test/core/utils/xhr-q.js

```diff
@@ -0,0 +1,108 @@
+/*global sinon */
+
+describe('axe.utils.xhrQ', function() {
```

Comment:
I think we need a few more tests here if we're going to bring in our own HTTP request thing. Consider how many tests are in https://github.com/axios/axios/tree/master/test vs here.

Reply:
See - #990, likely we will get rid of `xhrQ`